cmd/service_bootstrap_desktop.go

```diff
@@ -132,6 +132,12 @@ func (di *Dependencies) bootstrapServiceOpenvpn(nodeOptions node.Options) {
 		}
 
 		proposal := openvpn_discovery.NewServiceProposalWithLocation(currentLocation, transportOptions.Protocol)
+
+		var portSupplier port.ServicePortSupplier = di.PortPool
+		if transportOptions.Port != 0 && locationInfo.PubIP == locationInfo.OutIP {
```

Comment:
I'm a bit surprised nobody noticed. This actually changes behavior in most cases (because most clients will have a router), and `openvpn.port` flag/option is ignored and thus it's always randomized (which causes problems for desktop clients, so for now we decided to use static port `1194` - passing it through options when starting service via tequilapi).

Why this check was needed: `locationInfo.PubIP == locationInfo.OutIP`? I can't find any useful info in commit messages. Any ideas, @zolia , @vkuznecovas ?

Reply:
`locationInfo.PubIP == locationInfo.OutIP` supposed to check if we are behind the NAT. 
If IP on the host and public IP is the same, we, probably, don't need to do any NAT traversal since IP is reachable without it.

Reply:
True, but I think we should respect user flags despite this.